config/internal/apiserver/deployment.yaml.tmpl

```diff
@@ -18,6 +18,54 @@ spec:
         component: data-science-pipelines
     spec:
       containers:
+        {{ if .APIServer.EnableRoute }}
+        - name: oauth-proxy
+          args:
+            - --https-address=:8443
+            - --provider=openshift
+            - --openshift-service-account=ds-pipeline-{{.Name}}
+            - --upstream=http://localhost:8888
+            - --tls-cert=/etc/tls/private/tls.crt
+            - --tls-key=/etc/tls/private/tls.key
+            - --cookie-secret=SECRET
+            - '--openshift-delegate-urls={"/": {"group":"route.openshift.io","resource":"routes","verb":"get","name":"ds-pipeline-{{.Name}}","namespace":"{{.Namespace}}"}}'
```

Comment:
couldn't quite tell from the docs -- `"/"` means "delegate authn/authz for everything, pass if the user has get access to the route in the namespace ... ?

Reply:
`/` refers to the path to which this rule applies to, you can for example restrict it to only paths under `/api`, the rule allows only people that have `get` on `routes` in the ns where this proxy resides, this is no different than how we handled the ui oauth proxy, we expect users with a ds-project to have this permission